source/js/jquery-sortable.js

```diff
@@ -401,10 +402,11 @@
       ) >= this.options.distance)
     },
     getPointer: function(e) {
-      var o = e.originalEvent || e.originalEvent.touches && e.originalEvent.touches[0]
+      var o = e.originalEvent,
+        t = (e.originalEvent.touches && e.originalEvent.touches[0]) || {};
       return {
-        left: e.pageX || o.pageX,
-        top: e.pageY || o.pageY
+        left: e.pageX || o.pageX || t.pageX,
+        top: e.pageY || o.pageY || t.pageY
```

Comment:
Having both `e` and `o` here is probably redundant, but I did not know what bug may have initially led to this solution.